app/config/ApplicationConfig.scala

```diff
@@ -23,10 +23,10 @@ import uk.gov.hmrc.play.bootstrap.config.{RunMode, ServicesConfig}
 @Singleton
 class ApplicationConfig @Inject()(configuration: Configuration, runMode: RunMode, servicesConfig: ServicesConfig) {
 
-  def baseUrl(serviceName: String) = {
-    val protocol = servicesConfig.getConfString(s"microservice.services.$serviceName.protocol", "http")
-    val host = servicesConfig.getString(s"microservice.services.$serviceName.host")
-    val port = servicesConfig.getString(s"microservice.services.$serviceName.port")
+  private def baseUrl(serviceName: String) = {
```

Comment:
probably you could've left this function in original form but default it to https and just add protocol=http to config like below

Reply:
Tried that but it failed.